Real_world_examples/Water_quality_suspended_matter.ipynb

```diff
@@ -0,0 +1,960 @@
+{
```

Comment:
<p>If <code>LYM7</code> isn't used in this notebook, maybe we can just remove it? </p><p> </p><p>Also, just a styling choice, but I prefer something like <code>\textrm{LYM7}</code> for multi-letter variable names.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
I am thinking it might be good to keep the function in case someone wants to do water quality analysis on Landsat 7?